load-generator/README.md

```diff
@@ -27,4 +27,9 @@
 ### X-Ray Trace Load Test Sample Command,
 ```
 ./gradlew :load-generator:run --args="trace -r=100 -u=localhost:55680 -d=xray"
+```
+
+### StatsD Metrics Load Test Sample Command,
+```
+./gradlew :load-generator:run --args="metric -r=100 -u=localhost:55680 -d=statsd"
```

Comment:
normally statsd use 8125 port, can we change it to 8125 in readme?

Reply:
Make sense.